code/config.ss

```diff
@@ -131,16 +131,16 @@
      ((slug . two-fer)
       (uuid . "3ecc2d1c-55e0-45c9-ba35-57d7d8cfd51e")
       (core . #f)
-      (unlocked-by)
-      (difficulty . 1)
+      (unlocked-by hello-world)
+      (difficulty . 3)
```

Comment:
I am basically ready to merge this, but I wonder if `two-fer` really warrants difficulty level `3`?

Reply:
It is quite a bit simpler than `scrabble-score`

Reply:
It's not clear how to do optional arguments in scheme, imo. That was the reason for the elevated difficulty

Reply:
In guile it requires looking up some baroque syntax, in chez it involves looking up `case-lambda`, or understanding how arguments are passed as lists

Reply:
In addition to that, it also involves string concatenation or formatting

Reply:
Okay, I can live with that 😃